doc/contributing/collaborator-guide.md

```diff
@@ -879,9 +879,13 @@ If you cannot find who to cc for a file, `git shortlog -n -s <file>` can help.
 
 * `confirmed-bug`: Bugs you have verified
 * `discuss`: Things that need larger discussion
+* `fast-track`: PRs that need to land faster - see
```

Comment:
I wonder if putting https://github.com/nodejs/node/labels/fast-track (ie `https://github.com/nodejs/node/labels/fast-track` into the markdown will automatically get the color

Reply:
@MoLow are we getting the color or the others but not for these?